generators/go/internal/generator/sdk/internal/http.go

```diff
@@ -23,6 +23,30 @@ func ResolveBaseURL(values ...string) string {
 	return ""
 }
 
+// ResolveEnvironmentBaseURL resolves the base URL from an environment struct
+// by looking up the field with the given name. This is used for multi-URL
+// environments where different endpoints use different base URLs.
+func ResolveEnvironmentBaseURL(environment interface{}, fieldName string) string {
```

Comment:
Is it possible to only include this function when relevant? I.e. only when it is used in the multi URL case?